.github/workflows/relay.yml

```diff
@@ -0,0 +1,80 @@
+name: Relay
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "relay/**"
+      - ".github/workflows/relay.yml"
+  pull_request:
+    branches:
+    paths:
+      - "relay/**"
+      - ".github/workflows/relay.yml"
+  workflow_dispatch:
+
+jobs:
+  build-test-publish:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./relay
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-relay-cache
+          key: ${{ runner.os }}-buildx-relay-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-relay-
+
+      - run: sudo df -h
+
```

Comment:
We no longer need to monitor the disk space, so those line can be removed (I removed them in `keep-core` and `keep-ecdsa` in recent PRs - not yet merged).

Reply:
Please see https://github.com/keep-network/tbtc/commit/724110da01c83b57569db5d7120132bd27732ce0